tests/test_cgnsutilities.py

```diff
@@ -0,0 +1,41 @@
+import os
+import unittest
+from cgnsutilities.cgnsutilities import readGrid, BC
+
+baseDir = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestGrid(unittest.TestCase):
+    def setUp(self):
+        self.grid = readGrid(os.path.abspath(os.path.join(baseDir, "../examples/717_wl_L2.cgns")))
+        return
+
+    def test_getTotalCellsNodes(self):
+        totalCells, totalNodes = self.grid.getTotalCellsNodes()
+        self.assertEqual(totalCells, 15120)
+        self.assertEqual(totalNodes, 18753)
+
+    def test_getWallCellsNodes(self):
+        nWallCells, nWallNodes = self.grid.getWallCellsNodes()
+        self.assertEqual(nWallCells, 756)
+        self.assertEqual(nWallNodes, 893)
+
+    def test_overwriteFamilies(self):
+        # Find a specific BC and overwrite the family
+        famFile = os.path.abspath(os.path.join(baseDir, "../examples/overwriteFamily_famFile"))
+        # Check the family before overwriting.
+        self.assertEqual(self.grid.blocks[0].bocos[0].family.strip().decode("utf-8"), "wall")
+        self.grid.overwriteFamilies(famFile)
+        self.assertEqual(self.grid.blocks[0].bocos[0].family, "wing1")
+        return
```

Comment:
Why do some of these have the `return` statement?

Reply:
No reason, I meant to remove them.